src/index.js

```diff
@@ -0,0 +1,167 @@
+        table.appendChild(tr);
+
+        title = document.createElement('th');
+        title.innerHTML = allProjs[index].list[i].title;
+        tr.appendChild(title);
+
+        desc = document.createElement('th');
+        desc.innerHTML = allProjs[index].list[i].desc;
+        tr.appendChild(desc);
+
+        dueDate = document.createElement('th');
+        dueDate.innerHTML = allProjs[index].list[i].dueDate;
+        tr.appendChild(dueDate);
+
+        priority = document.createElement('th');
+        priority.innerHTML = allProjs[index].list[i].priority;
+        tr.appendChild(priority);
+
+        status = document.createElement('th');
+        if (allProjs[index].list[i].status) {
+          status.innerHTML = `<input type="checkbox" 
+                               id="taskStatus" 
+                               checked>`;
+        } else {
+          status.innerHTML = `<input type="checkbox" 
+                               id="taskStatus">`;
+        }
+        status.addEventListener('click', () => { updateListStatus(index, i); });
+        tr.appendChild(status);
+      }
+      div.appendChild(table);
+    }
+    return div;
+  }
+  return 0;
+}
+
+function displayTaskWithout() {
+  const div = document.getElementById('displayTaskW');
+  div.innerHTML = '';
+
+  const div2 = document.createElement('div');
+  const h2 = document.createElement('h2');
+  h2.innerHTML += getProjects()[0].title;
+  div2.appendChild(h2);
+  div2.appendChild(displaytask(0));
+  div.appendChild(div2);
+}
+
+function getTaskInter(index) {
+  getTaskData(index);
+  // eslint-disable-next-line no-restricted-globals
+  location.reload();
+}
+
+function opentaskModel(index = 0) {
+  document.getElementById('tskModel').style.display = 'grid';
+  document.getElementById('projModel').style.display = 'none';
+  document.getElementById('submitTask').addEventListener('click', () => { getTaskInter(index); });
+}
+
+function displayProject() {
+  const div = document.getElementById('displayProject');
+  div.innerHTML = '';
+
+  for (let i = 1; i < getProjects().length; i += 1) {
+    const div2 = document.createElement('div');
+    div2.setAttribute('class', 'projectsTasks mb-3');
+    const h2 = document.createElement('h2');
+    h2.innerHTML += getProjects()[i].title;
+    div2.appendChild(h2);
+    div2.appendChild(displaytask(i));
+
+    const button = document.createElement('button');
+    button.innerHTML = 'add task';
+    button.setAttribute('id', `add-task${i}`);
+    div2.appendChild(button);
+    div.appendChild(div2);
+
+    document.getElementById(`add-task${i}`).addEventListener('click', () => { opentaskModel(i); });
+  }
+}
+
+function getProjectsInter() {
+  getProjectsData();
+  // eslint-disable-next-line no-restricted-globals
+  location.reload();
+}
+
+function openProjectModel() {
+  document.getElementById('projModel').style.display = 'grid';
+  document.getElementById('tskModel').style.display = 'none';
+}
+
```

Comment:
if most of this pure DOM functions were assembled into a DOM module and called here (using a syntax [similar to something like this](https://github.com/shloch/js-module-template/blob/master/template_module.js)), this will bring some more organisation and this file `index.js` will breath  😊